lib/geordi/commands/unit.rb

```diff
@@ -5,7 +5,7 @@ def unit
     invoke_cmd 'yarn_install'
 
     Interaction.announce 'Running Test::Unit'
-    Util.system! 'bundle exec rake test'
+    Util.system! Util.binstub('rake', 'test')
```

Comment:
I would prefer invocation as `Util.system! Util.binstub('rake'), 'test'`, because only `rake` is the binstub here and `test` can be handled by `system!`.